web/themes/custom/customtheme/templates/block/block--customtheme-customweatherblock.html.twig

```diff
@@ -51,11 +53,11 @@
      %}
    </div>
     <p class="weather_content">
-      {{ content['#temp'] }}
+      {{ temp }}
     </p>
     <span class="celsium">
       c
```

Comment:
Don't understand the line C? Or the temperature or the letter, what is it?

Reply:
Better to use some glyph for C